wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/devfile/server/DevfileManager.java

```diff
@@ -163,6 +163,7 @@ public WorkspaceConfigImpl createWorkspaceConfig(
    * @throws ServerException when other error occurs
    * @see WorkspaceManager#getByKey(String)
    */
+  @SuppressWarnings("JavadocReference")
```

Comment:
I do :) The `@see` references a private method in `WorkspaceManager` which IJ complains about for me.

Reply:
Then maybe it would be better to fix an issue instead of suppress warning? 
The referenced private method tells nothing about a format, it contains code that parse key, but not java doc at all. From Java Doc perspective it would be better to reference 
```
public WorkspaceImpl getWorkspace(String key)
```
where key rules are described.
WDYT?

Reply:
:+1: I didn't change the method, so I didn't want to change the javadoc either and frankly didn't look into the method itself. Will change it.